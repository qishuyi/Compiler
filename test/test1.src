if 10 <= 8 then 8 / 2 + 6 else 2 * 6 - 10