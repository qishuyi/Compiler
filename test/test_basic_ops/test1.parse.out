(if (<= 10 8) (+ (/ 8 2) 6) (- (* 2 6) 10))
